src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -34,6 +37,7 @@ public ModelManager(ReadOnlyAddressBook addressBook, ReadOnlyUserPrefs userPrefs
         this.addressBook = new AddressBook(addressBook);
         this.userPrefs = new UserPrefs(userPrefs);
         filteredPersons = new FilteredList<>(this.addressBook.getPersonList());
+        eventList = FXCollections.observableArrayList();
```

Comment:
I think for this, it may be worth checking with @limweiliang. Our design was to have an `EventBook` like `AddressBook`, and the `EventBook` would encapsulate the actual `eventList`, and everything `Event` related would interact with the `ModelManager` object via the `EventBook` API.

Also just want to check - is the `observableArrayList` type safe? Don't see the `<>`.

Reply:
I shall defer to @limweiliang for this and other issues related to events. This particular implementation of events was a barebones skeleton/stub which I had created to get some kind of backend up for testing my GUI changes.